src/Collector/Stack.php

```diff
@@ -277,6 +287,38 @@ public function setResponseCode($responseCode)
         $this->responseCode = $responseCode;
     }
 
+    /**
+     * @return string
+     */
+    public function getDebugToken()
```

Comment:
can you please add `: ?string` instead of the `@return` annotation?

Reply:
Right now this is in line with the other properties. How about we keep it as is and instead I will do a follow up PR that adds type hints and return types for all properties on `Stack` to not mix styles?